### DISCUSSION TITLE
Screen widget not displaying anything

### DISCUSSION THREAD

**Discussion by guillermoventura** [Textualize/textual#3176] [Q&A] [answered]
When running an app and running install_screen it shows up and empty screen with no data. If I yield it all interactivity such as writing on inputs has some sort of delay or is not shown on screen. I ran one of your sample codes on your webpage to verify if it was the syntax or I was missing something but the same error happened.

```python
from textual.app import App, ComposeResult
from textual.screen import Screen
from textual.widgets import Static


ERROR_TEXT = """
An error has occurred. To continue:

Press Enter to return to Windows, or

Press CTRL+ALT+DEL to restart your computer. If you do this,
you will lose any unsaved information in all open applications.

Error: 0E : 016F : BFF9B3D4
"""


class BSOD(Screen):
    BINDINGS = [("escape", "app.pop_screen", "Pop screen")]

    def compose(self) -> ComposeResult:
        yield Static(" Windows ", id="title")
        yield Static(ERROR_TEXT)
        yield Static("Press any key to continue [blink]_[/]", id="any-key")


class BSODApp(App):
    CSS_PATH = "screen02.css"
    BINDINGS = [("b", "push_screen('bsod')", "BSOD")]

    def on_mount(self) -> None:
        self.install_screen(BSOD(), name="bsod")


if __name__ == "__main__":
    app = BSODApp()
    app.run()
```

No errors just an empty screen.

<img width="1279" alt="image" src="https://github.com/Textualize/textual/assets/38434061/9418bc44-863a-4f7d-80e4-3a6ded8dc816">

**Comment by willmcgugan** [comment, marked as answer]
Did you press `b` ?

`install_screen` doesn't change the current screen. It just creates a [named screen](https://textual.textualize.io/guide/screens/#named-screens)

**Comment by guillermoventura** [comment]
I needed to pop the new screen, got a bit confused with the documentation. I am trying to build a TUI Frontend and this tool is the one I liked the most but having a hard time to figure out some things.

**Comment by willmcgugan** [comment]
No worries. Join our [discord server](https://discord.gg/Enf6Z3qhVr). We're happy to help.
